README.md

```diff
@@ -1,2 +1,69 @@
 # newsweek-clone
 A replica of the www.newsweek.com website for www.microverse.org html and css week 3 project module
+
+### Mobile version
+
+![screenshot](./DesktopVersion.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML5
+- CSS3
+- Bootstrap5 Beta
+
+
+## Live Demo
+
+[Live Demo Link](https://oxenprogrammer.github.io/newsweek-clone/)
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Setup
+Checkout the project on your web browser with the provided link.
+
+### Install
+- Clone it
+- run `npm install`
+- run `gulp start` 
```

Comment:
We don't need it

Reply:
Hiii!! Thanks for the review. In the original, some of these texts overflow...it all depends on the text at the time, it is a dynamic website. We've really done our best please. Thanks alot